terraform/bin/deploy.sh

```diff
@@ -2,4 +2,4 @@
 set -eo pipefail
 
 source "$(dirname $BASH_SOURCE)/config.sh"
-terraform apply
+terraform apply -auto-approve
```

Comment:
The `-auto-approve` isn't needed here because the CI/CD pipeline uses a different command line